src/pkg/cli/client/byoc/aws/byoc.go

```diff
@@ -604,6 +604,16 @@ func (b *ByocAws) GetServices(ctx context.Context, req *defangv1.GetServicesRequ
 		listServiceResp.Project = projUpdate.Project
 	}
 
+	lbName := fmt.Sprintf("defang-%s-%s-alb", req.Project, b.PulumiStack) // same as shared/common.ts
```

Comment:
No we shouldn't have to do this. `ProjectUpdate` should already have it: https://github.com/DefangLabs/defang/blob/f909caba8c07ef2f649cbf6583a22c66c1643ec4/src/protos/io/defang/v1/fabric.proto#L434

Reply:
oh, we forget to set it in the TS code.

Reply:
Should we set it and use it when present, and fall back to this name approach if missing?

Reply:
why would it be missing? Existing projects? I wouldn't worry about them.

Reply:
oh sorry, that's the ARN, we need dnsName.